pom.xml

```diff
@@ -22,7 +22,7 @@
     <parent>
         <artifactId>open-source-parent</artifactId>
         <groupId>com.basistech</groupId>
-        <version>2.0.3</version>
+        <version>7.0.1</version>
```

Comment:
7.0.1? (ta 63.0.1) Why?

Reply:
I can't go any higher until the new ADM release.  The OSGI test fails.

Reply:
Waiting for new ADM to go higher. Thanks Seth!